apis/beacon/blocks/blinded_block.yaml

```diff
@@ -0,0 +1,61 @@
+get:
+  operationId: getBlindedBlock
+  summary: Get blinded block
+  description: |
+    Retrieves blinded block for given block ID.
+    Depending on `Accept` header it can be returned either as JSON or as bytes serialized by SSZ
+  tags:
+    - Beacon
+  parameters:
+    - name: block_id
+      in: path
+      required: true
+      $ref: '../../../beacon-node-oapi.yaml#/components/parameters/BlockId'
+  responses:
+    "200":
+      description: "Successful response"
+      headers:
+        Eth-Consensus-Version:
+          $ref: '../../../beacon-node-oapi.yaml#/components/headers/Eth-Consensus-Version'
+      content:
+        application/json:
+          schema:
+            title: GetBlindedBlockResponse
+            type: object
+            properties:
+              version:
+                type: string
+                enum: [phase0, altair, bellatrix]
```

Comment:
```suggestion
                enum: [bellatrix]
```

Reply:
I think it makes sense to return full phase0 and altair blocks on this endpoint as if they are blinded. It enables tools to just use a single endpoint without switching URLs at the Bellatrix fork.

There's a precedent for doing this already with the [blinded block production endpoint](https://ethereum.github.io/beacon-APIs/?urls.primaryName=dev#/Validator/produceBlindedBlock), so I imagine all clients can implement it easily:

> Pre-Bellatrix, this endpoint will return a BeaconBlock.

Reply:
Sounds good!